### PR TITLE
-login/registraton checks

### LINKED ISSUE #20: Sprint 1: Registration check
**Sprint 1**: Current registration doesn't have any checks to make sure the application is complete or even if the email/password meets the basic criteria.
Make it so it checks for:
email address is the right format "__@__.__"
first and last names are the right format (use regex)
password contains at least 6 characters
All fields are filled out
email address doesn't already exist (if it does, alert if it's pending or approved)

### PR DESCRIPTION
**Pull request by ivelmiskin3**
-approved pending added to auth
-Admin: edit pending users

### CODE REVIEW

File: Components/Admin/PendingUsers.js [hunk 1]
```diff
@@ -63,22 +81,124 @@ class PendingUsers extends Component {
...
+  async edit(pendingUser) {
+    this.setState({
+      editable: !!(this.state.editable % 2) ? pendingUser.editable : pendingUser.editable + 1,
+      currentUser: pendingUser,
+      currentCommittee: pendingUser.Committee
+    })
+    Alert.alert("You may now edit the user")
+  }
+
+  async submit(pendingUser) {
```
Comment (Tonewall) [inline Components/Admin/PendingUsers.js:93]: Not submitting the user to firebase.auth

**Review by Tonewall — COMMENTED** [review]
User is not added to authentication

File: Components/Admin/PendingUsers.js (hunk 1 above)
Comment (ivelmiskin3) [reply Components/Admin/PendingUsers.js:93]: approved User gets added for the first time in the checkUsers() method in Login.js

Comment (Tonewall) [reply Components/Admin/PendingUsers.js:93]: got it, it's working 👍

**Review by Tonewall — APPROVED** [review]

**Comment by Tonewall** [comment]
Just going to merge it since no one wants to review it

**Pull request merged**